app/pages/Home.css

```diff
@@ -46,11 +46,16 @@
 
       & .title {
         color: white;
-        font-size: 46px;
+        font-size: 40px;
+        font-style: normal;
         font-weight: 600;
+        line-height: 130%;
```

Comment:
I know I'm not the most consistent with this, but I think we should start using "[unitless line heights](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)", which helps with nesting. Unitless values for line-height mean "multiply the font-size by this much". I've had problems with percentage line heights in the past.

I think this line should be `line-height: 1.3;`, but I'm not sure if percentages actually map correctly to line-height (I've seen percentage values map to width and not height in most cases with CSS).